reference/cookie_consent.yml

```
@@ -0,0 +1,94 @@
+openapi: 3.0.1
+servers:
+  - url: 'https://{$$.env.store_domain}/api/storefront'
+info:
+  title: Storefront Cookie Consent API
```

```suggestion
  title: Storefront Cookie Consent
```
Remove "API" to match the rest of the API Reference documentation.